test/board_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/ship.rb'
+require './lib/cell.rb'
+require './lib/board.rb'
+
+class BoardTest < Minitest::Test
+  def setup
+    @board = Board.new
+  end
+
+  def test_it_exists
+    assert_instance_of Board, @board
+  end
+
+  def test_board_has_cells_as_hash
+    assert_equal Hash, @board.cells.class
+  end
+
+  def test_is_validates_coordinates
+    assert_equal true, @board.valid_coordinate?("A1")
+    assert_equal true, @board.valid_coordinate?("D4")
+    assert_equal false, @board.valid_coordinate?("A5")
+    assert_equal false, @board.valid_coordinate?("E1")
+    assert_equal false, @board.valid_coordinate?("A22")
```

Comment:
These files looks great - and so clean! Especially like how you've asserted multiple true and false values in test_is_validates_coordinates